src/components/ImageView/index.js

```diff
@@ -129,22 +129,11 @@ class ImageView extends PureComponent {
      * @param {Number} imageHeight
      */
     setImageRegion(imageWidth, imageHeight) {
-        let width = imageWidth;
-        let height = imageHeight;
+        const width = imageWidth;
+        const height = imageHeight;
         const containerHeight = this.state.containerHeight;
         const containerWidth = this.state.containerWidth;
 
-        // return if image not loaded yet
-        if (imageHeight <= 0 || containerHeight <= 0) {
-            return;
-        }
-
-        // Fit the image to container size if image small than container.
-        const aspectRatio = Math.min(containerHeight / imageHeight, containerWidth / imageWidth);
-        if (aspectRatio > 1) {
-            width *= (aspectRatio);
-            height *= (aspectRatio);
-        }
```

Comment:
Why are we removing this code?

Reply:
this code scale image to container height (143-147). When image comes smaller than the modal.

Reply:
ok, but I think this code is necessary. 

```
   if (imageHeight <= 0 || containerHeight <= 0) {
            return;
        }
```

Reply:
My first offer was as follows.

```js
if (imageHeight <= 0) {
            return;
        }
```
ContainerHeight may not have arrived when the image is loaded. That's why I suggested a change like this before.

You are right. If the image height is not reached, the continuation of the code does not matter. It won't be able to set the image height anyway.

I will add line like this.
```js
if (imageHeight <= 0) {
            return;
        }
```
I am making new commit now. Thank you.